contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -51,6 +51,14 @@ def testConstruction( self ) :
 		s = IECoreScene.SceneInterface.create( fileName, IECore.IndexedIO.OpenMode.Read )
 		self.assertEqual( s.fileName(), fileName )
 
+	def testMissingFileThrowsException( self ) :
+
+		fileName = os.path.dirname( __file__ ) + "/data/cubeMissing.usda"
+
+		with self.assertRaises(RuntimeError) as cm:
```

Comment:
Tiny nitpick here - by assigning to `cm` you're implying it is going to be used for something, but it isn't.